server/reader/index.js

```diff
@@ -0,0 +1,41 @@
+var cassandra = require('./node_modules/cassandra-driver');
+var client = new cassandra.Client({ contactPoints: ['172.31.14.84'], keyspace: 'events' });
+var QUEUE_URL = 'https://sqs.us-east-2.amazonaws.com/331983685977/packaged-events';
+var AWS = require('aws-sdk');
+var sqs = new AWS.SQS({region : 'us-east-2'});
+
+module.exports.handler = function(eventRecord) {
+  var query = 'SELECT * FROM Events.event WHERE videoid= ? AND viewinstanceid = ? allow filtering;';
+  var queryParams = [eventRecord.videoId, eventRecord.viewInstanceId];
```

Comment:
Is it necessary to search based on video id and view instance id? The view instance id should be unique, so it doesn't seem necessary.

Reply:
I was not clear on if the view instance ID is unique. I cannot remember which service is issuing it so I didn't want to assume it was always unique.

Reply:
I clarified this with Dono. He's not finalized the ID, but it probably will be unique. I will update with a new commit soon.

Reply:
Closing this to repull....